### ISSUE TITLE
spell checker

### ISSUE THREAD

**Issue by jerabaul29** [onivim/oni2#2481]
It does not seem that spell checker is available / enabled (?). What is the recommended way to get it enabled / should it be part of the 'core' of onivim2?

**Comment by bryphe** [comment]
Good question @jerabaul29 !

We could consider bundling the Vim dictionaries for spellcheck - but in the meantime, this VSCode extension _should_ work: https://open-vsx.org/extension/streetsidesoftware/code-spell-checker

However, when I tested it - I saw an error:

![image](https://user-images.githubusercontent.com/13532591/94196396-255b8f00-fe69-11ea-8a26-bd72f22bbeeb.png)

Looking into it to see if it's a quick fix, so that we can have some solution for spell-check in place.

**Comment by jerabaul29** [comment]
Ok, thanks.

I think it would be very useful with an 'official, well tested, integrated to Onivim2' solution. The idea is that if I want to spend lots of time on my settings I would use NeoVim or SpaceVim, would be great to have something that is high quality and 'just works' here :) .

**Comment by bryphe** [comment]
Definitely!  It seems like the vscode extension has had significant investment (and in fact, Microsoft's own spell-check solution was deprecated in favor of it: https://github.com/microsoft/vscode-spell-check). So I think this extension will be a good starting point. Will be interested to hear how it works or if there any pieces missing from your workflow once it's unblocked.

...I also need to use it to prevent typos in the docs! 😬
